cypress/integration/pages/mediaAssetPage/testsForCanonicalOnly.js

```diff
@@ -41,7 +42,94 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         }
       });
     });
+    describe('Most Watched component', () => {
+      before(() => {
+        cy.getToggles(config[service].name);
+      });
+      // Allows constructing most watched path with variant
+      let serviceVariant = variant;
+      serviceVariant =
+        serviceVariant === 'default'
+          ? (serviceVariant = '')
+          : (serviceVariant = `/${variant}`);
+
+      const mostWatchedPath = `/${config[service].name}/mostwatched${serviceVariant}.json`;
+      it('should show/not show the Most Watched component if the toggle is enabled for the service', function test() {
+        // Find if the service has component enabled in toggles
+        cy.fixture(`toggles/${config[service].name}.json`).then(toggles => {
+          const mostWatchedIsEnabled = path(
+            ['mostPopularMedia', 'enabled'],
+            toggles,
+          );
+          cy.log(`Service toggle enabled ${mostWatchedIsEnabled}`);
```

Comment:
Do we still want all the logs in these tests?  Not sure if these were added just for debugging, or they're still useful anyway

Reply:
I thought they could be useful as there are a few reasons the component may or may not show, but seeing as we might only look at them if there is a failing test (which hopefully they won't be!), maybe we will never really look at these.